invenio_rdm_records/oai.py

```diff
@@ -26,9 +26,9 @@
 from .services.pids.providers.oai import OAIPIDProvider
 
 
-def dublincore_etree(pid, record):
+def dublincore_etree(pid, record, **serializer_kwargs):
     """Get DublinCore XML etree for OAI-PMH."""
-    json = DublinCoreXMLSerializer().dump_obj(record["_source"])
+    json = DublinCoreXMLSerializer(**serializer_kwargs).dump_obj(record["_source"])
```

Comment:
So if the second part of the tuple has the shape `{"schema_context": {...}}`, it will be passed to the schema as `context` here via the serializer (see the added passing of `**options` in `invenio_rdm_records/resources/serializers/dublincore/__init__.py` below).